packages/nextjs/app/page.tsx

```diff
@@ -11,6 +12,7 @@ const Home = () => {
           <span className="block text-4xl font-bold">Scaffold-Stark 2</span>
         </h1>
         <ConnectedAddress />
+        <Address address="0x074a19c75acc3e5947f8515c025bc4286b56572c763acec95198fc87d1df9895"></Address>
```

Comment:
Why do we have this hardcoded value here ? @metalboyrick

Reply:
This is my address to test the PFP, this page.tsx was uploaded unintentionally, it should be reset.

Reply:
was debugging, forgot to remove